magento/magento2
# Bug/ac 12310 product image hide csv import

…pdate through csv importer issue fix


### Description (*)
hide_from_product_page not updating in product update through csv importer

### Fixed Issues (if relevant)
- Fixes #38882

### Manual testing scenarios (*)

1. Add hide_from_product_page column for any of the product image in csv file, after the import, open the product in admin and  see if the image is hidden or not
2. Remove any of the image from hide_from_product_page column and see if the hide_from_product_page attribute is removed



### Contribution checklist (*)
